tests/functional/test_freeze.py

```diff
@@ -265,9 +266,13 @@ def test_freeze_bazaar_clone(script, tmpdir):
 
     """
 
+    src_path = os.path.join(
+        os.path.dirname(__file__),
+        '..', 'data', 'repos', 'bzr', 'django-wikiapp', 'release-0.1'
+    )
+
```

Comment:
I've not looked at this in context, but should this not use the test data directory fixture ```data```?